Makefile

```diff
@@ -26,16 +26,10 @@ TARGET_OS ?= linux
 BUILDX_BUILD_ARGS = --build-arg TARGETOS=$(TARGET_OS)
 
 # Build a specific maistra image. Example of usage: make maistra-builder_2.3
-# This target calls the multi target if the version is >= 2.5
+# Only build single arch image by using the build command
 ${BUILD_IMAGE}_%:
-	if [ $(firstword $(subst ., ,$*)) -ge 2 -a $(word 2, $(subst ., ,$*)) -ge 5 ]; then \
-		echo "Building multi-platform image"; \
-		$(MAKE) $@_multi; \
-	else \
-		echo "Building single-platform image with $(CONTAINER_CLI)"; \
-		$(CONTAINER_CLI) build -t ${HUB}/${BUILD_IMAGE}:$* \
-				 -f docker/$@.Dockerfile docker; \
-	fi
+	echo "Building single-platform image with $(CONTAINER_CLI)"; \
+	$(CONTAINER_CLI) build -t ${HUB}/${BUILD_IMAGE}:$* -f docker/$@.Dockerfile docker; \
```

Comment:
we don't need the `\` at the end

Reply:
Done, also added another change to call from push target directly the multi-target, so, no change is needed from the release repository in the jobs

Reply:
so it means that if I run make push locally it will always use the multi target? Why? What if I want to build locally and push to my personal registry?

Reply:
Tested locally: https://quay.io/repository/frherrer/maistra-builder/manifest/sha256:f87916030a1ef155703fe18c0824876f0c469d0f3bb8c4b926d6fdeef5d2480b